htdocs/resources/assets/less/app.less

```diff
@@ -0,0 +1,3 @@
+
+@import "../../../node_modules/bootstrap-less/bootstrap/bootstrap";
```

Comment:
Import css as less otherwise we get missing resource errors. Probably it doesn't merge the files properly